### PR TITLE
Feature/restaurant

### PR DESCRIPTION
**Pull request by Dipeshtwis**
What does this PR do:

- Use of webpack to bundle JavaScript application.
- Use tabbed browsing by navbar items
- Use of different modules

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,9 +4,29 @@ import menu from './menu';
...
+main.appendChild(home());
+parent.appendChild(main);
+
+const tab = (lid) => {
+  let a;
+  if (lid === 'home') a = home();
+  if (lid === 'menu') a = menu();
+  if (lid === 'contact') a = contact();
+
+  return a;
```
Comment (lucasmazo23) [inline src/index.js]: [OPTIONAL] I think you could use more descriptive variable names here.

**Review by lucasmazo23 — CHANGES_REQUESTED** [review]
## Changes required 🔧
### You are doing a great job so far, and the project is looking good.

### For the project to get approved kindly implement these changes:

### Required changes:
- [x] [REQUIRED] The link to the live version doesn't leave to anywhere (this was resolved while making the review)

- [x] [REQUIRED] Modules should be able to append themselves to the webpage

image

[source](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)


- [x] [OPTIONAL] I think you could use more descriptive variable names (check my comment below for more info).
### Happy coding! 💻
## Lucas Mazo

File: src/index.js [hunk 2]
```diff
@@ -4,9 +4,29 @@ import menu from './menu';
...
 
 parent.appendChild(nav());
-parent.appendChild(home());
-parent.appendChild(menu());
-parent.appendChild(contact());
+const link = document.querySelectorAll('.links');
+const main = document.createElement('div');
+
+main.appendChild(home());
+parent.appendChild(main);
```
Comment (lucasmazo23) [inline src/index.js]: this is what I was talking about when I said modules should be able to append themselves, this logic should be inside of each module (sorry I forgot to point to the lines where it was happening in my review)

File: src/index.js (hunk 1 above)
Comment (Dipeshtwis) [reply src/index.js]: done

**Comment by Dipeshtwis** [comment]
## The updated project includes:

- Updated Live link
- Now module directly appended to the root itself

**Review by bellom — APPROVED** [review]
## Hi @Dipeshtwis  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**